benchmarks/baselines/2026-03-03-scarb-baseline.md

```
@@ -0,0 +1,20 @@
+# Scarb Baseline Benchmark (20260303-032209)
+
+## Environment
+- Generated at: 2026-03-03T09:23:03Z
+- Matrix: research
+- Host: TMWM-G5FXKKXLXY
+- Tool: scarb 2.14.0 (682b29e13 2025-11-25)
+- Workspace root: /Users/espejelomar/StarkNet/compiler-starknet
+
+## Summary
+| Scenario | Workload | Runs | p50 (ms) | p95 (ms) | mean (ms) | min (ms) | max (ms) |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)